lib/bundle/commands/install.rb

```diff
@@ -1,7 +1,9 @@
 module Bundle::Commands
   class Install
     def self.run
-      Bundle::Dsl.new(Bundle.brewfile).install
+      success = Bundle::Dsl.new(Bundle.brewfile).install
+      exit 1 unless success
+      success
```

Comment:
How about `Bundle::Dsl.new(Bundle.brewfile).install || exit 1`?

IMHO, it's pointless to return `success` here. As it will always be true.